src/hydra/dataProvider.ts

```diff
@@ -806,6 +806,7 @@ function dataProvider(
         if (sub.count <= 0) {
           if (sub.subscribed && sub.eventSource && sub.eventListener) {
             sub.eventSource.removeEventListener('message', sub.eventListener);
```

Comment:
I'm wondering if this line is useful, WDYT?

Reply:
I'm guessing that the object should fall out of scope (no longer have any references) and will be removed after this (cleaned up by the garbage collector), which would make it not very useful. Maybe the original idea behind it was that the EventSource would be re-used by the browser instead of re-connecting and setting it up again, but I can't be sure.
I suppose it won't hurt to remove the event listener before closing though.